### PR TITLE
Small language fix

### PR DESCRIPTION
**Pull request by ikurecic**

### CODE REVIEW

**Review by AndrewGardhouse — APPROVED** [review]

File: demonstrations.rst [hunk 1]
```diff
@@ -106,7 +106,7 @@ Demos
                                     <br>
                                     <strong>Quantum Computing</strong>
                                     </h3>
-                                    <p class="mb-1 grey-text px-3">Explore general quantum computing tasks and algorithms, from quantum volume to boson sampling.</p>
+                                    <p class="mb-1 grey-text px-3">Explore general quantum computing tasks and algorithms, from measuring quantum volume to boson sampling.</p>
```
Comment (josh146) [inline demonstrations.rst:109]:
```suggestion
                                    <p class="mb-1 grey-text px-3">Explore general quantum computing concepts and algorithms, from quantum volume to boson sampling.</p>
```

**Review by josh146 — CHANGES_REQUESTED** [review]
Thanks Ivana! One small change, to keep the existing flow of the sentence

**Review by josh146 — APPROVED** [review]
💯

**Review by jen-dodd — APPROVED** [review]
Looks good!

**Pull request merged**